code/game/objects/items/weapons/switchtool.dm

```diff
@@ -59,6 +62,9 @@
 		choose_deploy(user)
 
 /obj/item/weapon/switchtool/attackby(var/obj/item/used_item, mob/user)
+	if(istype(used_item, /obj/item/weapon/screwdriver) && no_remove)
```

Comment:
I felt it was needed because people may think its a bug that you can't remove things form a switchtool like normal.